File: 백준/Silver/1822. 차집합/차집합.js
const input = require("fs")
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split("\n")

const [a,b] = input[0].split(' ').map(Number)
const A = new Set(input[1].split(' ').map(Number))
const B = new Set(input[2].split(' ').map(Number))

for(let i of B){
    A.delete(i)
}
if(A.size === 0){
    console.log(0)
}else{
    console.log(A.size)
    console.log( ...[...A].sort((a,b)=>a-b))
}

File: 백준/Silver/1822. 차집합/차집합.py
import sys
input = sys.stdin.readline

a,b = map(int,input().split())
A = set(map(int,input().split()))
B = set(map(int,input().split()))

answer = list(A.difference(B))
print(len(answer))
if(len(answer) != 0):
  answer.sort()
  print(*answer)